server/testutils/factories/arrival.ts

```diff
@@ -5,7 +5,7 @@ import type { Arrival } from '@approved-premises/api'
 import { DateFormats } from '../../utils/dateUtils'
 
 export default Factory.define<Arrival>(() => {
-  const arrivalDate = faker.date.soon()
+  const arrivalDate = faker.date.past()
```

Comment:
I think generally arrivals should be expected to be in the future. If they need to be in the past for special cases they can be overridden in the consuming code.